portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Task {
+
+  private final long id;
+  private final String title;
+  private final long timestamp;
+
+  public Task(long id, String title, long timestamp) {
+    this.id = id;
```

Comment:
Having the id be specified by the creating client is typically dangerous; you want IDs to be created by your internal system so that malicious clients can't spoof your object. You've got the right idea though, of assigning the value of the id at constructor time.

Reply:
Is there a tutorial over doing that? I just followed the example given by the walkthrough.

Reply:
Ah sorry, I don't get notifications about the comments either! Github...

Staying with the walkthrough is fine - we can talk about that in our session tomorrow :)